README.md

```diff
@@ -1,44 +1,60 @@
 [![Build Status](https://travis-ci.org/cloudfoundry-incubator/eirini.svg?branch=master)](https://travis-ci.org/cloudfoundry-incubator/eirini)
 
-# Eirini - what this?
+# What is Eirini?
 
-Eirini is a Kubernetes backend for Cloud Foundry. It deploys CF apps to a kube backend, using OCI images and Kube deployments.
+*Eirini* is a Kubernetes backend for Cloud Foundry.
+It deploys CF apps to a kube backend, using OCI images and Kube deployments.
 
-_But there's more!_
+Eirini exports staged CF images as docker images.
```

Comment:
This is no longer true; the bits-service handles that.

Reply:
sure I can remove it. I asked this question on slack on whether it makes sense to have the registry definition back with a slight twist on how it is replaced by bit services: https://cloudfoundry.slack.com/archives/C8RU3BZ26/p1546897685022500?thread_ts=1546897659.022400&cid=C8RU3BZ26

but if it makes more sense to remove it, I can nuke it.

Reply:
Thanks and sorry, I did not tie the two things together. Yes, mentioning the role of the bits-service and pointing to it makes perfect sense.